.circleci/config.yml

```diff
@@ -0,0 +1,163 @@
+        type: string
+      filename:
+        type: string
+    steps:
+      - attach_workspace:
+          at: .
+      - wp-product-orb/variable_load
+      - run:
+          name: "Unzip plugin files"
+          command: |
+            cd ~/project
+            mkdir build
+            curl -sL https://github.com/wpengine/wp-graphql-content-blocks/releases/latest/download/wp-graphql-content-blocks.zip > <<parameters.slug>>.zip
+            unzip -o -d <<parameters.slug>> <<parameters.slug>>.zip
+            ls -laR
+      - wp-product-orb/get_version_from_php:
+          filename: <<parameters.slug>>/<<parameters.filename>>
+          return_var: BUILD_VERSION
+      - wp-product-orb/variable:
+          var: BUILD_VERSION
+          value: $BUILD_VERSION
+      - run:
+          name: "Move zip file to build directory"
+          command: |
+              mv <<parameters.slug>>.zip build/<<parameters.slug>>.$BUILD_VERSION.zip
+      - run:
+          name: "DEBUG"
+          command: |
+            ls -laR
+      - persist_to_workspace:
+          root: .
+          paths:
+            - .
+
+  plugin-build-json:
+    executor: wp-product-orb/parser
+    environment:
+      WPE_SESSION_DIR: ./.wpe
+    parameters:
+      slug:
+        type: string
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          command: |
+            cd ~/project
+      - wp-product-orb/variable_load
+      - wp-product-orb/parse_wp_readme:
+          infile: <<parameters.slug>>/readme.txt
+          outfile: build/<<parameters.slug>>.$BUILD_VERSION.json
+      - store_artifacts:
+          path: build
+      - run:
+          name: "DEBUG"
+          command: |
+            pwd
+            ls -laR
+      - persist_to_workspace:
+          root: .
+          paths:
+            - build
+
+  plugin-deploy:
+    executor: wp-product-orb/authenticate
+    environment:
+      WPE_SESSION_DIR: ./.wpe
+    parameters:
+      auth_url:
+        type: string
+      upload_url:
+        type: string
+      slug:
+        type: string
+    steps:
+      - attach_workspace:
+          at: .
+      - wp-product-orb/variable_load
+      - wp-product-orb/authenticate:
+          user: WPE_LDAP_USER
+          pass: WPE_LDAP_PASS
+          url: <<parameters.auth_url>>
+      - run:
+          name: "DEBUG"
+          command: |
+            pwd
+            ls -laR
+      - wp-product-orb/post_zip:
+          url: <<parameters.upload_url>>/<<parameters.slug>>
+          zip: build/<<parameters.slug>>.$BUILD_VERSION.zip
+          json: build/<<parameters.slug>>.$BUILD_VERSION.json
+          version: $BUILD_VERSION
+
+workflows:
+  deploy:
+    jobs:
+      - plugin-unzip:
+            slug: wpgraphql-content-blocks
+            filename: wp-graphql-content-blocks.php
+            # Run this job when a tag is published.
+            filters:
+              branches:
+                ignore: /.*/
+              tags:
+                only: /^v.*/
```

Comment:
I've updated all these to run directly in CircleCI when tags are created, instead of using the GHA to trigger Circle. I ran into some known bugs with the Circle trigger action. This also simplifies things.